### ISSUE TITLE
Arrow keys for selecting options doesn't work in Windows

### ISSUE THREAD

**Issue by nmai** [SBoudrias/Inquirer.js#303]
This issue is directly related: https://github.com/eslint/eslint/issues/3880

Pressing the arrow keys does nothing. Running node.js v4.1.2 under Windows 8.1.

**Comment by SBoudrias** [comment]
Duplicate of https://github.com/SBoudrias/Inquirer.js/issues/266 - you have the full background of the issue there and Node.js fixed the issue on their master branch.

**Comment by nmai** [comment]
@SBoudrias Thank you, and my apologies for creating a duplicate. I upgraded to Node 5.0.0 in hopes that it would have the latest patch. It did not. Hopefully they'll roll out the fix in the coming days.

I discovered that the arrow keys worked perfectly fine in https://github.com/Maximus5/ConEmu but the default Windows command prompt, Node.js command prompt, and Git command prompt all failed. 

**tl;dr:** The only solution to this problem is to use ConEmu or generate the `.eslintrc` file yoursef, at least until a fix is released by the Node team.

Or just use a better shell environment, such the ones offered by Linux and OS X :+1:

**Comment by Spaider** [comment]
For some reason, ConEmu does not always work as well. I did not investigate it a lot, but it worked for me only for non-admin console.

**Comment by Amberlamps** [comment]
I am running Node in 5.0.0 on Windows 8.1. Arrow keys do not work in this simple example:

```
var questions = [
  {
    type: "list",
    name: "color",
    message: "Pick a color",
    choices: ['red', 'green', 'blue']
  }
];

inquirer.prompt(questions, (answers) => {
  console.log(answers);
});
```

But when I add a simple boolean it magically works. Maybe this sheds a little light on this issue.

```
var questions = [
  {
    type: "confirm",
    name: "mood",
    message: "Are you feeling alright?",
    default: true
  },
  {
    type: "list",
    name: "color",
    message: "Pick a color",
    choices: ['red', 'green', 'blue']
  }
];

inquirer.prompt(questions, (answers) => {
  console.log(answers);
});
```

**Comment by SBoudrias** [comment]
@Amberlamps please see related issues. It's all explained in deep. Node release a bug causing a race condition who's not creating keypress events on windows.

**Comment by geddski** [comment]
Here's a workaround for anyone that need it. https://gist.github.com/geddski/c42feb364f3c671d22b6390d82b8af8f

**Comment by Jeff-Lewis** [comment]
Also see https://github.com/nodejs/node/issues/5384

**Comment by jokeyrhyme** [comment]
I have a module that helps identify broken versions of Node:

- https://github.com/jokeyrhyme/is-node-interactive-tty-broken

Do you think it might be worth integrating a similar check into this library? So that a warning or something could be displayed? Or perhaps modifying the "engines" part of package.json to help avoid broken Node versions at installation time?

**Comment by amartiuk** [comment]
Keypress doesn't work in Git Bash for me also. But it works fine with ConEmu64 (https://conemu.github.io/) on Windows 10.
